Tools/clinic/clinic.py

```diff
@@ -850,6 +878,58 @@ def render(
                 function = o
         return self.render_function(clinic, function)
 
+    def deprecate_positional_use(
+            self,
+            func: Function,
+            params: dict[int, Parameter],
+    ) -> str:
+        assert len(params) > 0
+        code_blocks: list[str] = []
```

Comment:
This variable is assigned to but never used

(This review comment comes courtesy of pyflakes)

```suggestion
```

Reply:
Yes, remnants from an experiment with multiple deprecation levels.